.husky/pre-commit

```diff
@@ -6,5 +6,5 @@ npx gulp lint-ts && \
 npx gulp scripts && \
 npx ts-node ./test/ts-node-unit-tests && \
 npx gulp test && \
-npx gulp test-ts && \
-npx gulp dashboards/test
+npx gulp test-ts # && \
```

Comment:
Do we need the commented line?

Reply:
We don't,  I've just left for the future, when the task with implementing conditional  pre-commit hooks is ready